File: examples/cle.basic/embed.call.other/files/AddFunction.py
SrvGroup = starpy._GetSrvGroup()
Service = SrvGroup._GetService("","")
Obj=Service._New("TestClass");
def Obj_Add(self,x,y) :
    return x+y+self.Value;  # Value is defined by caller
Obj.Add = Obj_Add;
Obj.ChildValue = 200;

File: trunk/examples/cle.basic/embed.call.other/files/AddFunction.lua
SrvGroup = libstarcore._GetSrvGroup()
Service = SrvGroup:_GetService("","")
Obj=Service:_New("TestClass");
function Obj:Add(x,y)
  return x+y+self.Value;  --Value is defined by caller
end
Obj.ChildValue = 200;